CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 - Added home path to logo and title
 - Added polygon configuration to chainbridgeConfig
 - Updated title
+- Configured destination chain as CERE
```

Comment:
Rename to `Set Cere Network as Destination Network`.

Reply:
Updated.